main.go

```diff
@@ -335,6 +335,9 @@ func findAPKsAndAABs(pths []string) (apks []string, aabs []string, others []stri
 func deploy(clearedFilesToDeploy []string, config Config) (ArtifactURLCollection, error) {
 	apks, aabs, others := findAPKsAndAABs(clearedFilesToDeploy)
 
+	log.Warnf(`Universal APK generation is going to be DEPRECATED with the next major release! Please use the Export Universal APK Step instead!
```

Comment:
Would mention in which use cases the Export step is needed

Reply:
Good idea, I will update the message with something like `in case you build a bundle and you would like to generate a universal APK from that`.

Reply:
Would mention that if they want to distribute the APKs for testing (by Public install page or Ship Add-on) then it is needed.

Reply:
Sure, I can tweak it.